src/components/home/TimeTracker.tsx

```diff
@@ -95,8 +121,8 @@ const TimeTracker = () => {
 
   return (
     <Container>
-      <BillableHours billable={data.billable} />
-      <GrowthHours growth={data.growth} />
+      <BillableHours billable={data.billable} dbUpdate={data.dbUpdate} />
```

Comment:
If you are changing the API according to my code review comments, this will need to be fixed